src/shelly_temp_sensor_ow.cpp

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (c) Shelly-HomeKit Contributors
+ * All rights reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "shelly_temp_sensor_ow.hpp"
+
+#ifdef MGOS_HAVE_ONEWIRE
```

Comment:
let's avoid conditional compilation via propocessor. instead, move this code into a subdirectory and only add it to the models that need it.